client/my-sites/upgrades/components/form/state-select.jsx

```diff
@@ -36,27 +36,24 @@ module.exports = React.createClass( {
 		} else {
 			this.refs.state.focus();
 		}
-	},
+	}
+
+	query() {
+		return this.props.countryCode && <QueryCountryStates countryCode={ this.props.countryCode } />;
```

Comment:
Usage seems a bit awkward. Maybe let's update QueryCountryStates to do a no-op if countryCode is falsey?